### DISCUSSION TITLE
Removing/Changing internally stored e-mail adresses

### DISCUSSION THREAD

**Discussion by Cerno-b** [WeblateOrg/weblate#10345] [Q&A]
I am using weblate hosted and I have worked on some translations, only to later realize that the e-mail address in my account informaton is extracted to the po files.

I used my personal e-mail address to create my account and now my real name is stored in the internals of Weblate, and on the next translation commit to the project I contributed to it will be immortalized in the project repo, where it will likely never be removed. I don't want my user name to be associated with my real name online.

Now I changed my e-mail address to an anonymous one, but now the po export contains two entries, one for my old e-mail and one for the new one.

**Question:** _Is it possible for an admin to remove one of the entries, so that the user/address combo will not be part of the export?_ 
Since I'm just a user, I have no access to the admin portal, so I can't check this myself.

Wouldn't it generally be a good idea to export e-mail addresses to po files in an anonymized way? j*.b*@gmail.com or something like this? Or make it configurable by the admins? Or make it so that each user has an opt-in option to use their e-mail address in the export?

I think this is a somewhat relevant privacy issue. Should I open an issue for this? 
Anyway for now, I just would like to know whether I can get rid of my e-mail address in the Weblate database.

**Comment by nijel** [comment]
As a user, you can change your email address in profile.  You can even use private one for committing,  see https://docs.weblate.org/en/latest/user/profile.html#profile

**Reply by Cerno-b** [reply]
@nijel Thanks but that's not my problem. I have modified my e-mail address, but now **both** are contained in the .po file.

Is this a bug? It seems like every time a contribution is made, the current e-mail address is added to the internal list of contributers, and when a user changes their address, this information is not *updated* but *added*. 

Not sure if this is the desired behavior, but what I don't like about it is that I as a user do not seem to have full control over my privacy information. And since po exports are often committed to a git repo, this can become a privacy nightmare, especially since it's not clearly communicated by Weblate what your e-mail address is used for.

Do you know if there is a way to retroactively remove one of the addresses?

**Reply by nijel** [reply]
It behaves as intended. Weblate cannot rewrite history in the external Git repository. But I've just written a longer reply to https://github.com/WeblateOrg/weblate/issues/10352 (it is really not useful to start the second place to discuss the same topic).

**Reply by Cerno-b** [reply]
Thanks, yeah, I probably shouldn't have, thanks for you time. I'll answer in the other issue.

**Comment by Cerno-b** [comment]
I wrote an issue #10352 at least for some added clarity on the "about" page.
